### PR TITLE
Update tests for PolicyInsights

### PR DESCRIPTION
**Pull request by j5lim**
Added a test for policyEvaluationDetails feature.
Updated session records with the correct version.

Related REST spec PRs:
https://github.com/Azure/azure-rest-api-specs/pull/5134
https://github.com/Azure/azure-rest-api-specs/pull/5408

### CODE REVIEW

File: src/SDKs/PolicyInsights/Management/Management.PolicyInsights/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Policy Insights Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure policy insights operations.")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.1.0")]
+[assembly: AssemblyVersion("2.0.2.0")]
```
Comment (dsgouda) [inline src/SDKs/PolicyInsights/Management/Management.PolicyInsights/Properties/AssemblyInfo.cs]: `AssemblyVersion` Remains the same, AssemblyFileVersion needs to be updated

**Review by dsgouda — CHANGES_REQUESTED** [review]

**Comment by dsgouda** [comment]
Verified tests work [here](https://azuresdkci.westus2.cloudapp.azure.com/view/NetCore-SDK/job/NetCore-SDK-Sign/524/console)

File: src/SDKs/PolicyInsights/Management/Management.PolicyInsights/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (j5lim) [reply src/SDKs/PolicyInsights/Management/Management.PolicyInsights/Properties/AssemblyInfo.cs]: Done.

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by j5lim** [comment]
@dsgouda Thanks for completing the PR. When will the new PolicyInsight nuget package (version 2.0.2) be released? Is there any process I need to follow to request a release?

**Comment by dsgouda** [comment]
@j5lim Please kick off a publish job using the instructions [here](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md) in order to publish the Nuget package.

**Pull request merged**